ehr/resources/queries/study/parentsWrongGender.sql

```diff
@@ -0,0 +1,15 @@
+SELECT * FROM (
+SELECT d1.Id,
+d1.Dataset.Demographics.gender,
+d1.Dataset.Demographics.species,
+d1.Parents.dam,
+d1.Parents.dam.Demographics.gender as damSex,
+d1.Parents.sire,
+d1.Parents.sire.Demographics.gender as sireSex,
+CASE WHEN (d1.Parents.dam.Demographics.gender.code IS NOT NULL AND d1.Parents.dam.Demographics.gender.code != 'f') OR (d1.Parents.sire.Demographics.gender.code IS NOT NULL AND d1.Parents.sire.Demographics.gender.code != 'm')
+THEN TRUE
+ELSE FALSE
+END as parentSexMismatch
+FROM study.Animal d1
```

Comment:
this is fine, but wouldnt it be a little better to just base this on study.demographicsParents rather than animal? you're filtering anything not present in animal, arent you?

Reply:
Ideally, yes it would be better, but not all centers have study.demographicsParents working. I'm honestly just trying to keep things simple. we can always improve on it later if necessary.

Reply:
Yeah it was my suggestion to use Animal. Only one center doesn't have demographicsParents, but this won't work for them anyway. We should probably look into bringing demographicsParents into core EHR and setting up the Parents column in DefaultEHRCustomizer. @jryurkanin we can sync on this.

Reply:
It seems like it might be useful to create a demographicsPedigree.sql in core EHR, and have this SQL select from study.demographics, and then merge in ehr.supplemental_pedigree. That would make columns consistent across EHR instances, and by default support supplemental_pedigree.

Reply:
I have something along those lines for the base demographicsParents in this PR. Most centers already have it overwritten, but we can use whatever version of demographicsParents is active for these queries.

https://github.com/LabKey/ehrModules/pull/541